examples/graph_visual_with_falkordb.py

```
@@ -0,0 +1,266 @@
+            print(f"Inserted edges {i+1} to {min(i + BATCH_SIZE_EDGES, len(edges))}")
+
+        print("Data insertion completed successfully!")
+        
+        # Print some statistics
+        node_count_result = graph.query("MATCH (n:Entity) RETURN count(n) AS count")
+        edge_count_result = graph.query("MATCH ()-[r:DIRECTED]-() RETURN count(r) AS count")
+        
+        node_count = node_count_result.result_set[0][0] if node_count_result.result_set else 0
+        edge_count = edge_count_result.result_set[0][0] if edge_count_result.result_set else 0
+        
+        print(f"Final statistics:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)